packages/shared/friends/utils.ts

```diff
@@ -132,3 +133,23 @@ export function validateFriendRequestId(friendRequestId: string, ownId: string)
 
   return friendRequestId.includes(ownId)
 }
+
+/**
+ * Check whether a user is blocked by the current user
+ * @param userId - the ID of the user to check
+ * @returns true if the user is blocked, false otherwise
+ */
+export function isBlocked(userId: string) {
+  // Get the user's profile
+  const profile = getCurrentUserProfile(store.getState())
+
+  // Check whether the user is blocked
+  if (profile?.blocked?.includes(userId)) {
+    return true
+  }
+  return false
```

Comment:
Avoid the if and return the boolean expression (or false otherwise).
```suggestion
  return profile?.blocked?.includes(userId) ?? false
```